specreduce/calibration_data.py

```diff
@@ -23,8 +23,8 @@
 ]
 
 """
-Make specreduce_data optional. If it's available, great and we can access its data via
-pkg_resources. If not, we'll fall back to downloading and optionally caching it using
+Make ``specreduce_data`` optional. If it's available, great and we can access its data via
+``pkg_resources``. If not, we'll fall back to downloading and optionally caching it using
```

Comment:
Most user wouldn't understand what "via pkg_resources" mean. Please explain what it is really doing.

Reply:
yeah, something like "python's built-in libraries" would be clearer.

Reply:
otoh, this is a docstring in the code and not user-facing documentation. that said, i have changed this to limit mention of `pkg_resources` to where it's actually used.